### PR TITLE
Reporters: Broken links fixed

### PR DESCRIPTION
**Pull request by MargaritaLoseva**
Broken links fixed.
Also, I've shortened long code lines in the code snippets.

/cc @inikulin @VasilyStrelyaev

### CODE REVIEW

File: docs/articles/documentation/extending-testcafe/custom-reporter/helper-methods.md [hunk 1]
```diff
@@ -326,10 +326,10 @@ moment().format() → this
 
 ```js
 reportTaskDone (endTime, passed) {
-    const durationMs  = endTime - this.startTime;
-    const durationStr = this.moment.duration(durationMs).format('h[h] mm[m] ss[s]');
+    const durMs  = endTime - this.startTime;
+    const durStr = this.moment.duration(durMs).format('h[h] mm[m] ss[s]');
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/extending-testcafe/custom-reporter/helper-methods.md]: Wouldn't it be better if we keep old names and just format it this way? @inikulin 
```
    const durationMs  = endTime - this.startTime;
    const durationStr = this.moment
                            .duration(durMs)
                            .format('h[h] mm[m] ss[s]');
```

File: docs/articles/documentation/extending-testcafe/custom-reporter/index.md [hunk 2]
```diff
@@ -127,13 +127,13 @@ reportTestDone (name, errs) {
 },
 
 reportTaskDone (endTime, passed) {
-    const durationMs  = endTime - this.startTime;
-    const durationStr = this.moment.duration(durationMs).format('h[h] mm[m] ss[s]');
-    let footer      = passed === this.testCount ?
-                      `${this.testCount} passed` :
-                      `${this.testCount - passed}/${this.testCount} failed`;
+    const durMs  = endTime - this.startTime;
+    const durStr = this.moment.duration(durMs).format('h[h] mm[m] ss[s]');
```
Comment (VasilyStrelyaev) [inline docs/articles/documentation/extending-testcafe/custom-reporter/index.md]: the same about this one

**Comment by VasilyStrelyaev** [comment]
\r-

**Comment by inikulin** [comment]
lgtm

**Comment by MargaritaLoseva** [comment]
fpr

**Comment by VasilyStrelyaev** [comment]
lgtm

**Comment by inikulin** [comment]
lgtm

**Pull request merged**